asset_dashboard/views.py

```diff
@@ -39,9 +41,37 @@ def get_context_data(self, **kwargs):
         # send the form to a modal in this view
         form = ProjectForm()
         context['form'] = form
+
+        # need all of the Sections and Categories for filtering the table
+        context['sections'] = [s.name for s in Section.objects.all().order_by('name')]
+        context['categories'] = [c.slug for c in ProjectCategory.objects.all().order_by('slug')]
+
         return context
 
 
+class ProjectListJson(BaseDatatableView):
+    model = Project
+    columns = ['name', 'description', 'section_owner', 'category', 'id']
+    order_columns = ['name', 'description', 'section_owner__name', 'category__category']
+    max_display_length = 500
+
+    def filter_queryset(self, qs):
+        search = self.request.GET.get('search[value]', None)
+        section = self.request.GET.get('columns[2][search][value]', None)
+        category = self.request.GET.get('columns[3][search][value]', None)
+
+        if search:
+            qs = qs.filter(Q(name__icontains=search) | Q(description__icontains=search))
+
+        if section:
+            qs = qs.filter(section_owner__name__icontains=section)
```

Comment:
this should be an exact match right? not an icontains?

Reply:
Good point. Fixed here: 423a709]